### PR TITLE
chore: standarize MFE i18n pattern

### PR DESCRIPTION
**Pull request by OmarIthawi**
Slight modifications to ensure it matches the other repos i18n structure.
    
   - use the standard src/i18n/transifex_input.json file path
   - use the standard path for messages: src/i18n/messages/
   - update i18n docs to use the centralized Open edX documentation
   - update Makefile to use the new paths
   - soft links are kept OEP-58 is fully implemented to avoid breaking components unintentionally

  

Refs: FC-0012 OEP-58

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @OmarIthawi!  Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

**Comment by OmarIthawi** [comment]
@mphilbrick211 this is tested on my fork and is ready for review and merge:

 - https://github.com/Zeit-Labs/openedx-translations/actions/runs/4435937895/jobs/7783752563

File: src/data/i18n/default/transifex_input.json
Comment (brian-smith-tcril) [inline src/data/i18n/default/transifex_input.json:1]: what is still relying on `transifex_input.json` existing in `src/data/i18n/default`?

it seems with the makefile changes we should be able to just remove this from here, but it's very possible i'm overlooking something

File: src/data/i18n/locales
Comment (brian-smith-tcril) [inline src/data/i18n/locales:1]: what is still relying on `locales` existing in `src/data/i18n`?

it seems with the makefile changes we should be able to just remove this from here, but it's very possible i'm overlooking something

**Comment by openedx-webhooks** [comment]
@OmarIthawi 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/studio-frontend&entry.867055334=https://github.com/openedx/studio-frontend/pull/374&entry.1484655318=https://github.com/OmarIthawi&entry.752974735=2023-03-07+16:43&entry.1917517419=2023-03-21+18:42&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**